content/guides/integrations/dropwizard.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Dropwizard"
+date: 2018-11-05T14:42:17-07:00
+draft: false
+class: "shadowed-image lightbox"
+aliases: [/integrations/dropwizard]
+weight: 5
+---
+
+- [Background](#background)
```

Comment:
I think we can reorganize this section to provide an example that works end-to-end
```shell
- [Background](#background)
- [Prerequisities](#prerequisites)
- [Adding dependences](#adding-dependencies)
- [Enable Dropwizard](#enable-dropwizard)
- [Enable an exporter](#enable-an-exporter)
- [End to end example](#end-to-end-example)
- [Visuals](#visuals)
```

where for:
* the end-to-end example we actually have a full pom.xml file because currently we aren't including the Prometheus imports which are just the exporters. Please make the end-to-end code copy and pastable from end to end otherwise people will struggle running it
* "Enable an exporter" which talk about enabling Prometheus but they can use any other exporter

Reply:
Done